1/Dockerfile

```diff
@@ -49,6 +49,9 @@ RUN /usr/local/bin/plugins.sh /opt/openshift/base-plugins.txt && \
     touch /opt/openshift/plugins/ssh-credentials.jpi.pinned && \
     touch /opt/openshift/plugins/script-security.jpi.pinned && \
     chown -R 1001:0 /opt/openshift && \
+    rmdir /var/log/jenkins && \
+    mkdir -p /var/lib/jenkins/logs && \
+    ln -sf /var/lib/jenkins/logs /var/log/jenkins && \
```

Comment:
i think we can't actually create the link here, because when you actually mount an empty volume into /var/lib/jenkins it's going to shadow the logs directory.

the symlink needs to be created in the same place where we populate the rest of /var/lib/jenkins (in the run script, during startup)